rubocop.yml

```diff
@@ -0,0 +1,1858 @@
+# Common configuration.
```

Comment:
tried to be consistent with Crowdtap style guide: https://crowdtap.atlassian.net/wiki/display/EN/Code+Style
Can definitely be revised.